shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java

```diff
@@ -662,6 +662,7 @@ protected void onNewIntent(@NonNull Intent intent) {
   }
 
   @Override
+  @SuppressWarnings("MissingSuperCall")
   public void onBackPressed() {
     flutterFragment.onBackPressed();
```

Comment:
Oh, I think what we should probably do is to actually call super.onBackPressed :)